ui/src/api/api.ts

```diff
@@ -1,5 +1,4 @@
 import axios from 'axios';
-console.log(process.env.REACT_APP_BASE_URL);
 const axiosApi = axios.create({
   baseURL: `${process.env.REACT_APP_BASE_URL}`,
```

Comment:
i'm not sure, i think only in create react app envs have to start by "REACT_APP", you have to check it